torch/nn/modules/module.py

```diff
@@ -1572,10 +1582,16 @@ def _call_impl(self, *args, **kwargs):
 
         return result
 
-    __call__ : Callable[..., Any] = _call_impl
+    __call__ : Callable[..., Any] = _wrapped_call_impl
```

Comment:
This does not look sound to inline. I think dynamo is choking on it in your tests. Discussing offline :)

Reply:
Ho I'm curious why? We are inlining `_call_impl` today. Why would inlining this new function not work?